core/store/src/opener.rs

```diff
@@ -448,7 +448,10 @@ impl<'a> StoreOpener<'a> {
             store.set_db_version(version + 1)?;
         }
 
-        if cfg!(feature = "nightly") || cfg!(feature = "nightly_protocol") {
+        if cfg!(feature = "nightly")
+            || cfg!(feature = "nightly_protocol")
+            || cfg!(feature = "statelessnet_protocol")
+        {
             let version = 10000;
```

Comment:
I'm afraid to ask what is going on here.

Reply:
From what I saw here, DB version is set to smth large if protocol is not stable.
Actually, now I'm not sure why this is needed at all. We may want to upgrade DB on master and then we'll want to see stateless nodes upgraded as well. So I'm removing this for statelessnet